lldb/source/Plugins/Process/Utility/RegisterInfoPOSIX_arm64.cpp

```diff
@@ -73,19 +73,20 @@
 #undef DECLARE_REGISTER_INFOS_ARM64_STRUCT
 
 static lldb_private::RegisterInfo g_register_infos_pauth[] = {
-    DEFINE_EXTENSION_REG(data_mask), DEFINE_EXTENSION_REG(code_mask)};
+    DEFINE_EXTENSION_REG(data_mask, KIND_ALL_INVALID),
+    DEFINE_EXTENSION_REG(code_mask, KIND_ALL_INVALID)};
 
 static lldb_private::RegisterInfo g_register_infos_mte[] = {
-    DEFINE_EXTENSION_REG(mte_ctrl)};
+    DEFINE_EXTENSION_REG(mte_ctrl, KIND_ALL_INVALID)};
 
 static lldb_private::RegisterInfo g_register_infos_tls[] = {
-    DEFINE_EXTENSION_REG(tpidr),
+    DEFINE_EXTENSION_REG(tpidr, GENERIC_KIND(LLDB_REGNUM_GENERIC_TP)),
```

Comment:
I think it would be simpler to just expand `DEFINE_EXTENSION_REG` just for tpidr, instead of changing it for every use. Only going to have one TLS pointer so it's justified I think.